src/open_inwoner/cms/cases/views/cases.py

```diff
@@ -20,11 +22,15 @@
 from open_inwoner.utils.mixins import PaginationMixin
 from open_inwoner.utils.views import CommonPageMixin
 
+from ..forms import CaseFilterForm
 from .mixins import CaseAccessMixin, CaseLogMixin, OuterCaseAccessMixin
 
 logger = logging.getLogger(__name__)
 
 
+SUBMISSION_STATUS_OPEN = _("Openstaande aanvraag")
```

Comment:
Translation in the template was indeed missing; I've added it.

Reply:
The statuses are retrieved in `openzaak/api_models.py` and indeed not translated, since we don't have control over what statuses are returned. We can translate the "status" for open submissions since it is our own. Do you think it would be better to leave it untranslated as well for consistency?